### ISSUE TITLE
Add live mode to livepeer_bench

### ISSUE THREAD

**Issue by yondonfu** [livepeer/go-livepeer#1699]
A user can configure the tool to simulate transcoding a live stream by transcoding a segment only after it would appear in a live playlist based on the segment's duration.

For consideration: the option to interleave segments from concurrent sessions i.e. seg1 from sess1 comes in, seg1 from sess2 comes in, seg2 from sess1 comes in as we suspect utilization will be higher in this case.

EDIT: Interleaving segments i.e. by staggering the start of streams is out of scope for this particular issue, but could be considered separately.
